modules/nw-multus-add-pod.adoc

```diff
@@ -7,17 +7,36 @@ ifeval::["{context}" == "configuring-sr-iov"]
 :sriov:
 endif::[]
 
+ifeval::["{product-version}" == "4.3"]
+:bz:
+endif::[]
+ifeval::["{product-version}" == "4.4"]
+:bz:
+endif::[]
+ifeval::["{product-version}" == "4.5"]
+:bz:
+endif::[]
+
 [id="nw-multus-add-pod_{context}"]
 = Adding a Pod to an additional network
 
 You can add a Pod to an additional network. The Pod continues to send normal
 cluster related network traffic over the default network.
 
+If a Pod already exists, you cannot attach it to an additional network. Only when a Pod is created are any additional networks requested attached to the Pod.
```

Comment:
To offset the first sentence, it might help to start the sentence, "However, if a Pod already exists...." 

Also, the next sentence is a bit hard to follow who is doing what to whom. If it's still accurate, maybe something like, "Additional requested networks are only attached when a Pod is created."

Reply:
@bobfuru, true, it doesn't really make that much sense as it is.

Reply:
@bobfuru might try

> When a Pod is created additional networks are attached to it. However, if a Pod already exists, you cannot attach additional networks to it.